### PR TITLE
Add debug information for Appfile configs

### PR DESCRIPTION
**Pull request by KrauseFx**
This will make it easier for people to detect if an Appfile was loaded, and what values were detected

screenshot 2016-10-06 18 07 12

### CODE REVIEW

File: credentials_manager/lib/credentials_manager/appfile_config.rb [hunk 1]
```diff
@@ -41,12 +43,35 @@ def initialize(path = nil)
...
+      self.class.already_printed_debug_information ||= {}
+      return if self.class.already_printed_debug_information[self.data]
+      # self.class.already_printed_debug_information is a hash, we use to detect if we already printed this data
+      # this is necessary, as on the course of a fastlane run, the values might change, e.g. when using
+      # the `for_lane` keyword.
+
+      puts "Successfully loaded Appfile at path '#{path}'".yellow
+
+      self.data.each do |key, value|
+        puts "- #{key.to_s.cyan}: '#{value.to_s.green}'"
```
Comment (asfalcone) [inline credentials_manager/lib/credentials_manager/appfile_config.rb]: Should we use the table that we use for tool params and plugins?

image 
image

Comment (KrauseFx) [inline credentials_manager/lib/credentials_manager/appfile_config.rb]: Great idea, I'll update the PR 👍

Comment (KrauseFx) [inline credentials_manager/lib/credentials_manager/appfile_config.rb]: Ah, the code is in _credentials_manager_, which has no dependency to `terminal-table`. I don't think it's worth adding the dependency for this

Comment (asfalcone) [inline credentials_manager/lib/credentials_manager/appfile_config.rb]: I guess this is only for verbose logging, to 👌

**Review by asfalcone — APPROVED** [review]

**Pull request merged**